ports/java/src/test/java/com/vmware/antlr4c3/TestCodeCompletionCore.java

```diff
@@ -153,8 +150,8 @@ public void test2_typicalExpressionTest() throws Exception {
         assertTrue(candidates.tokens.containsKey(ExprLexer.VAR));
         assertTrue(candidates.tokens.containsKey(ExprLexer.LET));
 
-        assertEquals(Arrays.asList(new Integer[]{ExprLexer.ID, ExprLexer.EQUAL}), candidates.tokens.get(ExprLexer.VAR));
```

Comment:
`ExprLexer.ID` and `ExprLexer.EQUAL` are members of the ignored token set, so they should not be candidate completions.